pkg/utils/utils.go

```diff
@@ -1300,29 +1263,9 @@ func GetNamespaces(ctx context.Context, ctrlClient crclient.Client) ([]string, e
 	return namespaces, nil
 }
 
-// CheckAccAndCreateOrDeleteRbac checks if the dell connectivity client exists and creates/deletes the role and rolebindings
-func CheckAccAndCreateOrDeleteRbac(ctx context.Context, operatorConfig OperatorConfig, ctrlClient crclient.Client, isDeleting bool) error {
-	logInstance := logger.GetLogger(ctx)
-	accList := &csmv1.ApexConnectivityClientList{}
-	if err := ctrlClient.List(ctx, accList); err != nil {
-		logInstance.Info("dell connectivity client not found")
-	} else if len(accList.Items) <= 0 {
-		logInstance.Info("dell connectivity client not found")
-	} else {
-		logInstance.Info("dell connectivity client found")
-		clientNamespace := accList.Items[0].Namespace
-		accConfigVersion := accList.Items[0].Spec.Client.ConfigVersion
-		brownfieldManifestFilePath := fmt.Sprintf("%s/clientconfig/%s/%s/%s", operatorConfig.ConfigDirectory,
-			csmv1.DreadnoughtClient, accConfigVersion, BrownfieldManifest)
-		if err = BrownfieldOnboard(ctx, brownfieldManifestFilePath, clientNamespace, ctrlClient, isDeleting); err != nil {
-			logInstance.Error(err, "error creating role/rolebindings")
-			return err
-		}
-	}
-	return nil
-}
-
+// 2nd parameter is an ApexCC cr
```

Comment:
Should this entire comment block be removed? It is related only to ACC, which we are trying to remove.

Reply:
yes, removed.